homeassistant/components/upnp/sensor.py

```diff
@@ -198,6 +198,8 @@ def state(self) -> str:
         """Return the state of the device."""
         device_value_key = self._sensor_type["device_value_key"]
         value = self._coordinator.data[device_value_key]
+        if value is None:
```

Comment:
Do you specifically need to compare against `None`? If not, this is shorter:

```python
if not value:
  return None
```

Reply:
Yes. `0` and `0.0` are valid values for this and the sensors below, respectivly. These sensors are counters which wrap around after 2^31-1 bytes. Although the value 0/0.0 is unlikely, certainly possible.